src/compute/Cargo.toml

```diff
@@ -7,6 +7,7 @@ rust-version.workspace = true
 publish = false
 
 [dependencies]
+ahash = { version = "0.8.0", default_features = false }
```

Comment:
I am a bit confused, likely due to the large diff and multiple revert/backport versions, so let me confirm this: In this version, we are _not_ introducing `mz_consolidate` methods and thus _not_ moving any `AHash` related code to the `mz_compute` crate. Is this correct? If so, this looks like a dependency that should remain associated to `timely-util`.

Reply:
Yes, that's correct. Currently, there is no added value of moving consolidate-related functions into rendering. Let me figure out what's going on here, I suspect it's a leftover from rebasing. Thank you!

Reply:
Perfect, thanks for the additional commit!